build.xml

```diff
@@ -1995,7 +1997,6 @@
                                              script.icon="DecoderPro"/>
         <create-macosx-application-directory script.name="PanelPro"      script.class="apps.PanelPro.PanelPro"/>
         <create-macosx-application-directory script.name="SoundPro"      script.class="apps.SoundPro.SoundPro"/>
-        <create-macosx-application-directory script.name="InstallTest"   script.class="apps.InstallTest.InstallTest"/>
```

Comment:
We have not included this in OS X / macOS since sometime before 4.0, so I'm no longer building it (as building an app bundle is now expensive).